### PR TITLE
WIP Put a recruitment message in the HTML

### PR DESCRIPTION
**Pull request by commuterjoy**
This is for Jay's graduate experiment.

### CODE REVIEW

File: views/front-page.html [hunk 1]
```diff
@@ -35,6 +35,32 @@ <h1 class='carousel-heading' id="editors-picks-title" aria-hidden="true">{{conte
...
+   F:::::::::::::::F           T:::::T        
+   F::::::FFFFFFFFFF           T:::::T        
+   F:::::F                     T:::::T        
+   F:::::F                     T:::::T        
+ FF:::::::FF                 TT:::::::TT      
+ F::::::::FF                 T:::::::::T      
+ F::::::::FF                 T:::::::::T      
+ FFFFFFFFFFF                 TTTTTTTTTTT  
+
+Graduate? Hello! You've been inquisitive enough to find a link to our graduate scheme application form :-
```
Comment (charypar) [inline views/front-page.html]: Was that meant to be a smiley at the end of the line?

Comment (commuterjoy) [inline views/front-page.html]: Not entirely sure - i'm sort of making it up as I go along :)

Comment (charypar) [inline views/front-page.html]: Fair enough :)) I thought I spotted a typo. Was so proud of myself...

**Comment by ironsidevsquincy** [comment]
Just on the front page?

**Comment by commuterjoy** [comment]
@ironsidevsquincy Yeah.

**Comment by ironsidevsquincy** [comment]
:+1:

**Comment by charypar** [comment]
@commuterjoy the designer side of me says "logo needs better kerning", but up to you really

**Pull request merged**